docs/resources/integration.md

```diff
@@ -0,0 +1,27 @@
+# Integration
+
+SignalFx supports integrations to ingest metrics from other monitoring systems, connect to Single Sign-On providers, and to report notifications for messaging and incident management.
```

Comment:
I think that it would be worth mentioning in the documentation that users must have a token with admin rights in order to be able to edit integrations.

Reply:
good catch, i'll add that